tests/csrf_tests/tests.py

```diff
@@ -295,6 +347,35 @@ def test_token_node_with_new_csrf_cookie(self):
         csrf_cookie = resp2.cookies[settings.CSRF_COOKIE_NAME]
         self._check_token_present(resp, csrf_id=csrf_cookie.value)
 
+    def test_cookie_not_reset_on_accepted_request(self):
+        """
+        The csrf token used in posts is changed on every request (although stays equivalent).
```

Comment:
Wrapped. Wasn't aware of this rule; where does it come from? It is not warned on by flake8.